app/scripts/metamask-controller.js

```diff
@@ -1956,7 +1962,7 @@ export default class MetamaskController extends EventEmitter {
     checkForMultipleVersionsRunning();
   }
 
-  getTransactionMetricsRequest() {
+  getTransactionEventRequest() {
```

Comment:
Linked the comment concerning the type name, but as we're gathering what the metrics listeners need specifically, is `getTransactionMetricsRequest` more explicit?

Reply:
All renaming is rolled back now [ea4d8df](https://github.com/MetaMask/metamask-extension/pull/21456/commits/ea4d8df9246e32b1dc6c9f79bbada3f2a409a781)